config_examples/config_full.example.json

```diff
@@ -177,5 +177,6 @@
     "strategy": "SampleStrategy",
     "strategy_path": "user_data/strategies/",
     "dataformat_ohlcv": "json",
-    "dataformat_trades": "jsongz"
+    "dataformat_trades": "jsongz",
+    "position_adjustment_enable": false
```

Comment:
I'm honestly not sure about this.

While it CAN be set via configuration - it really shouldn't - as this is really a strategy property, which should be set in the strategy.
I think we have a few others of these (which could be set in config but are not listed here intentionally).